lib/VCard.ts

```diff
@@ -286,6 +286,16 @@ ${lastName};${firstName};${additional};${prefix};${suffix}\
     return this
   }
 
+  /**
+   * Add nickanem
```

Comment:
**typo:** `nickanem` instead of `nickname`.

```suggestion
   * Add nickname
```

Reply:
🤦